.github/workflows/build.yml

```diff
@@ -100,12 +100,12 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-python@v5
         with:
           python-version: "3.10"
-          cache: "pip"
-          cache-dependency-path: ".github/workflows/build.yml"
-      - run: pip install -U laminci
+      - run: |
+          python -m pip install -U uv
```

Comment:
Now you don't need this additional line anymore, @Zethson

Reply:
This will spare us headache in undoing calling "uv" out explicitly once it becomes the default resolver for pip (I assume this might not even be so far out).